include/mbgl/style/expression/error.hpp

```diff
@@ -10,17 +10,17 @@ namespace expression {
 
 class Error : public Expression {
 public:
-    Error(std::string message_)
+    Error(std::string message_) noexcept
```

Comment:
The move constructor of `std::string` is not `noexcept`.

Reply:
I usually rely on cppreference.com, is there somewhere else I should be looking?

![image](https://github.com/maplibre/maplibre-native/assets/71895881/12f03584-9c2f-4e61-af56-35347fca0ab7)

Reply:
Oh I was looking at the one with a custom allocator, my bad!